src/changes/changes.xml

```diff
@@ -6,6 +6,9 @@
     </properties>
     <body>
         <release version="2.3" date="in GIT" description="Maintenance">
+            <action dev="asomov" type="fix" issue="1087">
```

Comment:
This change (15fe5f6ecaa970805086fa31a7794ae1e0b9a196) isn't part of my PR. I cloned from bitbucket and it looks like the GitHub mirror is one commit behind. Updating the mirror shall cleanup my PR as well.

Reply:
Thank [you](https://github.com/snakeyaml/snakeyaml/pull/12#issuecomment-2108059560). Now my proposed changes are clearly visible in the _Files changed_ tab.